src/app/pages/faq/faq.component.html

```diff
@@ -54,17 +54,23 @@ <h2 class="mb-0">
         <h2 class="mb-0">
           <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#s1q3c"
                   aria-expanded="false" aria-controls="s1q3c">
-            How can I sign someone's virtrolio more than once?
+            Can I sign someone's virtrolio more than once?
           </button>
         </h2>
       </div>
 
       <!-- Answer -->
       <div id="s1q3c" class="collapse" aria-labelledby="s1q3h" data-parent="#faq-s1">
         <div class="card-body card-align">
-          Unfortunately, you can't. You probably wouldn't sign someone's yearbook more than once in real life (unless
+          Yes and no. For any one version of the website (for example, if you just created an account with us), then you
+          will only be able to send one message to each of your friends. You probably wouldn't
+          sign someone's yearbook more than once in real life
+          (unless
           you just <em>wanted</em> to take space away from others). As such, we decided that you can send one very
-          special, meaningful message to each of your friends. Make it count!
+          special, meaningful message. However, if we make a major update to the website,
+          then we will allow for <strong>one extra message to be sent to each of your friends</strong> so that you can
+          try out the new update's
+          features.
```

Comment:
Unnecessary line break
```suggestion
          try out the new update's features.
```